File: 프로그래머스/lv0/120884. 치킨 쿠폰/치킨 쿠폰.scala
object Solution {
    def solution(Chicken: Int): Int = {
        var chicken=Chicken :Int
        var answer=0:Int;
        while(chicken>=10){
            var tmp=chicken/10:Int
            answer+=tmp;
            chicken%=10;
            chicken+=tmp;
        }
        return answer;
    }
}

File: 프로그래머스/lv0/120884. 치킨 쿠폰/치킨 쿠폰.kt
class Solution {
    fun solution(Chicken: Int): Int {
        var chicken:Int = Chicken
        var answer: Int = 0
        while(chicken>=10){
            var tmp:Int=chicken/10
            answer +=tmp
            chicken%=10
            chicken+=tmp
        }
        return answer
    }
}